onnx/backend/test/case/node/sequenceinsert.py

```diff
@@ -0,0 +1,56 @@
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+from __future__ import unicode_literals
+
+import numpy as np  # type: ignore
+
+import onnx
+from ..base import Base
+from . import expect
+
+
+def sequence_insert_reference_implementation(sequence, tensor, position=None):  # type: (np.ndarray, np.ndarray, np.ndarray) -> np.ndarray
+    if position:
+        # In these cases, insert_position will be between [-len(sequence), len(sequence)]
+        # The position argument will be in the format np.array([pos_index])
+        insert_position = position[0]
+        sequence.insert(insert_position, tensor)
+    else:
+        # Default position of insertion is at the end of the sequence.
+        sequence.append(tensor)
+    return np.array(sequence)
```

Comment:
this sets output to `ndarray`, which is later recognized as tensor. Should be list for sequence.